pages/resilience.js

```diff
@@ -17,6 +18,9 @@ function resilience() {
           </p>
         </div>
       </TextAndPic>
+
+
+      <CovidBanner />
```

Comment:
Why so much empty space lol

Reply:
Do you wanna create an issue by referencing this comment so you don't forget? @LemonFace0309

Reply:
Lol idk, I guess I was anticipating other components to go there